packages/core-config/__tests__/__stubs__/network.json

```diff
@@ -27,9 +27,9 @@
       "epoch": "2017-03-21T13:00:00.000Z",
       "fees": {
         "dynamic": false,
+        "minFeePool": 1000,
+        "minFeeBroadcast": 1000,
         "dynamicFees": {
-          "minFeePool": 1000,
-          "minFeeBroadcast": 1000,
```

Comment:
It is wrong to have a notion of "minimum fee" outside of dynamic fees config. Minimum fee only makes sense in the context of dynamic fees.

In static fees the transaction fee must exactly match the configured static fee, there is no minimum or maximum.

Given the above, I think it is most natural for a static-fee configured node to do that - reject transactions whose fee do not match a pre-configured one. This is also what v1 does.